html/controls/operator.js

```diff
@@ -726,154 +739,92 @@ function createTeamTable() {
+			.attr("id", "Team"+team+"Lost").addClass("KeyControl").button().appendTo(leadJammerTd);
+		WSActiveButton(prefix + ".Lead", $("<button>")).text("Lead")
+			.attr("id", "Team"+team+"Lead").addClass("KeyControl").button().appendTo(leadJammerTd);
+		WSActiveButton(prefix + ".Calloff", $("<button>")).text("Call")
+			.attr("id", "Team"+team+"Call").addClass("KeyControl").button().appendTo(leadJammerTd);
+		WSActiveButton(prefix + ".Injury", $("<button>")).text("Inj")
+			.attr("id", "Team"+team+"Inj").addClass("KeyControl").button().appendTo(leadJammerTd);
+		WSActiveButton(prefix + ".NoInitial", $("<button>")).text("NI")
+			.attr("id", "Team"+team+"NI").addClass("KeyControl").button().appendTo(leadJammerTd);
+
 		leadJammerTd.buttonset();
 
 		var starPassTd = jammer2Tr.children("td:eq("+(first?"0":"1")+")");
-		var starPass = sbTeam.$sb("StarPass");
-		var starPassButton = starPass.$sbControl("<button>").text("Star Pass").val("true")
-			.attr("id", "Team"+team+"StarPass").addClass("KeyControl").button();
-		starPass.$sbBindAndRun("sbchange", function(event, value) {
-			starPassButton.val(!isTrue(value));
-			starPassButton.toggleClass("Active", isTrue(value));
-		});
-		starPassButton.appendTo(starPassTd);
-		var noPivot = sbTeam.$sb("NoPivot");
-		var noPivotButton = starPass.$sbElement("<button>").text("No Pivot").val("true")
-			.attr("id", "Team"+team+"NoPivot").addClass("KeyControl").button();
-		noPivot.$sbBindAndRun("sbchange", function(event, value) {
-			noPivotButton.val(!isTrue(value));
-			noPivotButton.toggleClass("Active", isTrue(value));
-		});
-		noPivotButton.appendTo(starPassTd);
+		var starPassButton = WSActiveButton(prefix + ".StarPass", $("<button>")).text("Star Pass")
+			.attr("id", "Team"+team+"StarPass").addClass("KeyControl").button().appendTo(starPassTd);
+		var noPivotButton = WSActiveButton(prefix + ".NoPivot", $("<button>")).text("No Pivot")
+			.attr("id", "Team"+team+"NoPivot").addClass("KeyControl").button().appendTo(starPassTd);
 
 		var makeSkaterDropdown = function(pos, elem, sort) {
-			var sortFunc = _windowFunctions.alphaCompareByProp;
-			if (sort == "Num") sortFunc = _windowFunctions.numCompareByProp;
-			return sbTeam.$sb("Position("+pos+").Skater").$sbControl("<select>", { sbelement: {
-					optionParent: sbTeam,
-					optionChildName: "Skater",
-					optionNameElement: elem,
-					optionFilterElement: "Role",
-					optionChildFilter: function(node) { return (node.$sb("Role").$sbGet() != 'NotInGame'); },
-					compareOptions: function(a, b) { return sortFunc("text", a, b); },
-					firstOption: { text: '?', value: "" }
-				} }).addClass(pos+" By"+elem+" "+sort+"Sort");
+			var select = $("<select>").append($("<option value=''>?</option>"));
+			WS.Register([prefix + ".Skater(*).Number", prefix + ".Skater(*).BaseRole"], function(k, v) {
+				select.children("[value='"+k.Skater+"']").remove();
+				if (v != null && WS.state[prefix + ".Skater("+k.Skater+").BaseRole"] == "Bench") {
```

Comment:
This will remove and thus unset a skater as they foul out. It is also inconsistent with all the other screens. Use `.Role != 'NotInGame'`.

Reply:
Done, plus fixed a small bug on the Teams tab.

Reply:
It should be `.Role`, not `.BaseRole`. If a Skater is marked as being in the current lineup when the Not Skating flag is set, we give precedence to the former everywhere else.

Reply:
Done